sopel/modules/meetbot.py

```diff
@@ -268,7 +271,7 @@ def chairs(bot, trigger):
         chairs_readable = trigger.group(2).lower().replace(' ', ', ')
         logplain('Meeting chairs are: ' + chairs_readable, trigger.sender)
         logHTML_listitem('<span style="font-weight: bold">Meeting chairs are: </span>' + chairs_readable, trigger.sender)
-        bot.say('Meeting chairs are: ' + chairs_readable)
+        bot.say(bold('Meeting chairs are: ') + chairs_readable)
```

Comment:
Here, as above, the space shouldn't be included in the bold format span if we're nitpicking.